docs_src/microservices/device/supported/device-usb-camera/walkthrough/setup.md

```diff
@@ -100,6 +100,36 @@ Clone the EdgeX compose repository:
 git clone https://github.com/edgexfoundry/edgex-compose.git
 ```
 
+## Proxy Setup (Optional)
```

Comment:
Pretty much same comments on the device-onvif-camera files applies here as well.